### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by Errolll** [zzangho/markdowneqnotion#1]

**Comment by zzangho** [comment]
1. Register notion API and get a notion API token from notion API center
2. Embed your registered API on the page you want to modify, you can embed it in the page options (can access via top right corner)
3. Get a page id by copying page url
4. Paste the API token (from 1) and the page id (from 3) onto the source code. they are annotated like (your api token here...) or (your page id here...)
5. install the required python package imported in the source code
6. run it and wait until the operation is finished

**Comment by Errolll** [comment]
<img width="1352" alt="Screenshot 2024-01-26 at 19 49 45" src="https://github.com/zzangho/markdowneqnotion/assets/126964351/97bb95cb-c675-48dc-b6a2-3f00d75f2daf">
Just like this? But where should I install the package to? And how to run it?

**Comment by zzangho** [comment]
No, you have to use your notion token. You can get it from your notion API center.
After you get the API token, please replace the annotated line in the source code with your API token.

in the main.py

```
import os
from notion_client import Client
from functions import convert_eq, convert_inline_eq

if __name__ == '__main__':
    
    # ex: os.environ["NOTION_TOKEN"] = "secret_***"
    os.environ["NOTION_TOKEN"] = "<YOUR NOTION TOKEN>"
    
    notion = Client(auth=os.environ["NOTION_TOKEN"])
    
    page_id = '<YOUR PAGE ID>'
    
    
    _blocks = list()
    
    has_more = True
    
    cursor_id = None

    ...
```

Replace `<YOUR NOTION TOKEN>`  with your notion API token and `<YOUR PAGE ID>` with your page ID.
, and run the main.py with the Python command.


This tutorial could help you understand how to run the Python code with notion API.

https://www.youtube.com/watch?v=M1gu9MDucMA
